actions.yaml

```diff
@@ -1,2 +1,88 @@
 run-migration:
   description: Run a command to create SQL schemas and apply migration plans.
+create-oauth-client:
+  description: Register an oauth client
+  params:
+    audience:
+      description: A list with the allowed audience for the client
+      type: array
+    grant-types:
+      description: A list with the allowed grant types for the client
+      type: array
+      default: [authorization_code]
+    redirect-uris:
+      description: A list with the client's redirect_uri
+      type: array
+    response-types:
+      description: A list with the allowed response types for the client
+      type: array
+      default: [code]
+    scope:
+      description: A list with the allowed scopes for the client
+      type: array
+      default: [openid, profile, email, phone]
+    client-secret:
+      description: The client's secret, if not provided one will be autogenerated.
+      type: string
+    token-endpoint-auth-method:
+      description: The authentication method the client may use at the token endpoint.
+      type: string
+get-oauth-client-info:
+  description: Get an oauth client's information
+  params:
+    client-id:
+      description: The client_id
+      type: string
+  required: ["client-id"]
+update-oauth-client:
+  description: Update an oauth client
+  params:
+    client-id:
+      description: The client_id
+      type: string
+    audience:
+      description: A list with the allowed audience for the client
+      type: array
+    grant-types:
+      description: A list with the allowed grant types for the client
+      type: array
+    redirect-uris:
+      description: A list with the client's redirect_uri
+      type: array
+    response-types:
+      description: A list with the allowed response types for the client
+      type: array
+    scope:
+      description: A list with the allowed scopes for the client
+      type: array
+    client-secret:
+      description: The client's secret, if not provided one will be autogenerated.
+      type: string
+    token-endpoint-auth-method:
+      description: The authentication method the client may use at the token endpoint.
+      type: string
+  required: ["client-id"]
+delete-oauth-client:
+  description: Delete an oauth client
+  params:
+    client-id:
+      description: The client_id
+      type: string
+  required: ["client-id"]
+list-oauth-clients:
+  description: List all oauth clients
+revoke-oauth-client-access-tokens:
+  description: Delete an oauth client's access tokens
+  params:
+    client-id:
+      description: The client_id
+      type: string
+  required: ["client-id"]
+rotate-key:
+  description: Rotate the jwk used for signing tokens
+  params:
+    alg:
+      description: The algorithm that should be used
+      type: string
+      default: RS256
+      enumerate: [RS256, RS512, ES256, ES512, EdDSA]
```

Comment:
Does Ory Ory support RS512? It's not mentioned in their [docs](https://www.ory.sh/docs/hydra/cli/hydra-keys-create)

Reply:
That's interesting, we use the [hydra create jwk](https://www.ory.sh/docs/hydra/cli/hydra-create-jwk) command which allows it.